homeassistant/components/flipr/select.py

```diff
@@ -0,0 +1,56 @@
+"""Select platform for the Flipr's Hub."""
+
+import logging
+
+from homeassistant.components.select import SelectEntity, SelectEntityDescription
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import FliprConfigEntry
+from .entity import FliprEntity
+
+_LOGGER = logging.getLogger(__name__)
+
+SELECT_TYPES: tuple[SelectEntityDescription, ...] = (
+    SelectEntityDescription(
+        key="hubMode",
+        name=None,
```

Comment:
I'd recommend to choose either the switch or the select to be the main feature I think?

Reply:
The switch is the main feature. So I set a name to the select.